components/src/core/hero/hero.component.ts

```diff
@@ -45,13 +51,21 @@ import { requireInput } from '../../utils/utils';
     },
 })
 export class HeroComponent implements OnChanges, AfterViewInit, AfterContentChecked {
+    /** Deprecated* Color of the hero icon */
     @Input() color: string;
-    @Input() label: string;
-    @Input() value: string;
-    @Input() units: string;
+    /** Deprecated* Color of the hero background */
+    @Input() iconBackgroundColor: string;
+    /** The size of the primary icon (10-48) */
     @Input() iconSize = 36;
+    /** The text shown below the Channel Value */
+    @Input() label: string;
+    /** Deprecated* Scales SVG icon */
```

Comment:
scales it how?

Reply:
Added some more info here; this is another prop that we talked about removing in the past. To make things simpler for now, I've removed all the deprecated tags since we don't have any planned actions to adjust these components.